visualization/app/codeCharta/util/settingsMerger.spec.ts

```diff
@@ -126,106 +126,40 @@ describe("SettingsMerger", () => {
 		})
 	})
 
-	describe("MarkedPackages merge", () => {
```

Comment:
Why were all these MarkedPackages tests deleted?

Reply:
Good catch. I can't explain why this happened. I re-added them